content/en/docs/contribute/localization.md

```diff
@@ -205,14 +205,21 @@ To ensure accuracy in grammar and meaning, members of your localization team sho
 
 ### Source files
 
-Localizations must be based on the English files from the most recent release, {{< latest-version >}}.
+Localizations must be based on the English files from a specific release targeted by the localization team.
+Each localization team can decide which release to target which is referred to as the _target version_ below.
 
-To find source files for the most recent release:
+To find source files for your target version:
 
 1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+2. Select branch for your target version from followings.
 
-The latest version is {{< latest-version >}}, so the most recent release branch is [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
```

Comment:
:information_source:  One thing I considered: setting up a GitHub Action to _mirror_ master into the release branch for the current release, and also blocking merges into that branch whilst it remains the mirror of the current release.

Reply:
I filed about reconsidering branch strategy into as issue #25806. Let's discuss at there.

Reply:
The discuss might take time. I'd like to describe about current workflow in this PR.